frontend/src/pages/Player/Toolbar/DevToolsWindowV2/NetworkPage/NetworkPage.tsx

```diff
@@ -102,6 +105,23 @@ export const NetworkPage = ({
 						requestType === RequestType.All ||
 						requestType === request.initiatorType,
 				)
+				.filter((request) => {
```

Comment:
There's a bug with how the request type and status interact with each other.
Here's a loom to describe the issue: https://www.loom.com/share/109f9d3fe14d493db7f97700ff8bb4aa

Happy to talk through how we might fix this.

Reply:
fixed this in #2398c86 thanks for catching this!